### PR TITLE
Flux 0.174 release notes 

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Flux release notes 0.174 https://github.com/influxdata/flux/releases/tag/v0.174.0

Small release, there wasnt many updates to this release. 

image

### CODE REVIEW

File: content/flux/v0.x/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,29 @@ aliases:
 ---
 
 
+## v0.174.0 [2022-07-05]
+
+### Features
+
+- Add coloring highlights to test outputs.
+- Promote [`experimental.to()`](/flux/v0.x/stdlib/experimental/to/) to 
+  [`influxdata.influxdb.wideTo()`](/flux/v0.x/stdlib/influxdata/influxdb/wideto/)
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:20]:
```suggestion
  [`influxdata.influxdb.wideTo()`](/flux/v0.x/stdlib/influxdata/influxdb/wideto/).
```

File: content/flux/v0.x/stdlib/experimental/to.md [hunk 2]
```diff
@@ -30,6 +31,9 @@ Fluxdoc syntax: https://github.com/influxdata/flux/blob/master/docs/fluxdoc.md
 
 `experimental.to()` writes _pivoted_ data to an InfluxDB 2.x or InfluxDB Cloud bucket.
 
+**Deprecated**: `experimental.to()` is deprecated in favor of [`wideTo()`](https://docs.influxdata.com/flux/v0.x/stdlib/influxdata/influxdb/wideTo/),
```
Comment (kelseiv) [inline content/flux/v0.x/stdlib/experimental/to.md:34]:
```suggestion
**Deprecated**: `experimental.to()` is deprecated in favor of [`wideTo()`](/flux/v0.x/stdlib/influxdata/influxdb/wideto/),
```

File: content/flux/v0.x/stdlib/influxdata/influxdb/wideto.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Wide data is _pivoted_ in that its fields are represented as columns making the table wider.
+
+#### Requirements and behavior
+- Requires both a `_time` and a `_measurement` column.
+- All columns in the group key (other than `_measurement`) are written as tags
+  with the column name as the tag key and the column value as the tag value.
+- All columns **not** in the group key (other than `_time`) are written as
+  fields with the column name as the field key and the column value as the field value.
+
+If using the `from()` to query data from InfluxDB, use pivot() to transform
```
Comment (kelseiv) [inline content/flux/v0.x/stdlib/influxdata/influxdb/wideto.md:42]:
```suggestion
If using the `from()` to query data from InfluxDB, use `pivot()` to transform
```

File: content/flux/v0.x/stdlib/experimental/catch.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: experimental.catch() function
```
Comment (kelseiv) [inline content/flux/v0.x/stdlib/experimental/catch.md:2]: is this a new function to mention in the release notes?

File: content/flux/v0.x/stdlib/testing/shoulderror.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+title: testing.shouldError() function
```
Comment (kelseiv) [inline content/flux/v0.x/stdlib/testing/shoulderror.md:2]: new function to mention--or something to keep behind the scenes?

**Review by kelseiv — APPROVED** [review]

**Comment by sunbryely-influxdata** [comment]
@kelseiv I need to edit documentation in the [Flux Repo](https://github.com/influxdata/flux) in order for the auto gen tool to make the requested changes. I'll do that right now and then regenerate the files.

**Comment by sunbryely-influxdata** [comment]
I don't see the new functions mentioned anywhere in Flux 0.174, but I'll comb through the commits to see where those changes are coming from and update the release notes accordingly

**Pull request merged**